### PR TITLE
Link the ScholarX Archive Page to the ScholarX 23 Page

### LINKED ISSUE #1551: Link ScholarX archive page to ScholarX page
**Is your feature request related to a problem? Please describe.**
We have to link the ScholarX archive page to the ScholarX page just like our past programs.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1551 

## Goals

Linking the ScholarX Archive Page to the ScholarX 23 Page

## Approach

I linked the ScholarX Archive Page to the ScholarX 23 Page

### Screenshots

image

  
### Preview Link

https://pr-1581-sef-site.surge.sh/scholarx/2023/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/2023/index.html [hunk 1]
```diff
@@ -205,6 +205,23 @@ <h3 class="mb-2 mt-4 text-onelive">Take part as a mentee under ScholarX23</h3>
   </div>
 </section>
 
+<!-- ScholarX Archive Section -->
+<section class="section">
+  <div id="past-onelives" class="container-fluid bg-white">
+    <div class="row justify-content-center">
+      <div class="col-9 text-center">
+        <h1 class="mb-0 text-onelive">Explore The Unveiling</h1>
```
Comment (anjula-sack) [inline scholarx/2023/index.html]:
```suggestion
        <h1 class="mb-0 text-onelive">Checkout our past mentors and mentees</h1>
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/2023/index.html [hunk 2]
```diff
@@ -205,6 +205,23 @@ <h3 class="mb-2 mt-4 text-onelive">Take part as a mentee under ScholarX23</h3>
   </div>
 </section>
 
+<!-- ScholarX Archive Section -->
+<section class="section">
+  <div id="past-onelives" class="container-fluid bg-white">
+    <div class="row justify-content-center">
+      <div class="col-9 text-center">
+        <h1 class="mb-0 text-onelive">Checkout our past mentors and mentees</h1>
+        <p class="display-3 text-onelive font-weight-bolder">ScholarX Archive</p>
```
Comment (anjula-sack) [inline scholarx/2023/index.html]:
```suggestion
        <p class="display-3 text-onelive font-weight-bolder">Checkout our past mentors and mentees</p>
```

Comment (anjula-sack) [reply scholarx/2023/index.html]: We don't need to mention archive part twice

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Let's break the title into two parts
image

just like in this section
image

**Comment by YoshithaRathnayake** [comment]
@anjula-sack how about now?

**Review by anjula-sack — APPROVED** [review]
Good job! @YoshithaRathnayake

**Pull request merged**